### ISSUE TITLE
Feature request: expose forwarded port via teleconsole.com

### ISSUE THREAD

**Issue by joeblew99** [gravitational/teleconsole#14]
I am using the golang tool Hugo to make static web sites.

But i need to allow third party servers to validate my code that is running locally.
For example:
https://validator.ampproject.org/#url=http%3A%2F%2Flocalhost%3A1313%2F

This validator is run by google, and wants to validate my html code thats running on my local web server. 

Another example is when I want to allow saucelabs access to my local web server, so thex can run my code on one of their mobile phones.

From what i can see teleconsole will not allow this to work at it is ?

BTW thanks for making this. Really useful.

Regards

joe

**Comment by kontsevoy** [comment]
@joeblew99 you are correct, this is not possible with teleconsole (it's an SSH client/server at its core)

**Comment by kontsevoy** [comment]
I won't be closing it because exposing a client's port via a server URL is possible in principle.
